example/bbs-dock.js

```diff
@@ -1,12 +1,13 @@
-import VerifiableCredential from '../src/verifiable-credential';
+import { initializeWasm } from '@docknetwork/crypto-wasm-ts';
 import { DIDResolver } from '../src/resolver';
 import Bls12381G2KeyPairDock2022 from '../src/utils/vc/crypto/Bls12381G2KeyPairDock2022';
+import { issueCredential, verifyCredential } from '../src/utils/vc';
 
 const keypairOpts = {
   id: 'did:example:489398593#keys-1',
   controller: 'did:example:489398593',
   privateKeyBase58: '23fKPbbSJ7tCJVDYynRZQ1wPs6GannD2dEMjKZnFtKxy',
-  publicKeyBase58: 'GzJYyMYXQyAdmRK3JcfQnHJtz5TyRiYyYtfj6SZD9LQ4zDapQ3qeC1AaBwCtzhQ8En6EGiLe7ke2RaeLNTxJ9GVompK6j6kHB8e5m2ya5qdCB2bbNMVii99QfmYsbj36J3B',
+  publicKeyBase58: 'przwNdX6Bn5TzwmX56fYKQr6vk5U2DsfJJHZJQzr1Sxd9oJUg1rEEoUP7Bz33WNpykvkkqoTByMwnceCx9yvTW8CG1V5XpSwHPSN222cwMe9xr4mViyLWkKtoraybEPeLHT',
```

Comment:
was this just incorrect before?

Reply:
Yes, used older (>1 year) format of keys.